### PR TITLE
Create oas.md

### PR DESCRIPTION
**Pull request by lizygeogy**

### CODE REVIEW

**Comment by lizygeogy** [comment]
@chinmaykolhatkar 
@sanjaypujare 
Please verify and merge this PR.

File: docs/oas.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Online Analytics Service (OAS)
+
+Online Analytics Service (OAS) is an Apex application which is delivered as a service in some of the DT RTS applications, for example, Omni Channel Fraud Prevention Application.  It is a Druid based system that can query in real-time on data streams from various source applications.
+
+OAS is integrated with OAS Dashboards service which is a DataTorrent RTS Service that has been built using Apache Superset. See [OAS Dashboards Service](oas_dashboards)
+
+OAS Dashboards performs visualization on the frontend, whereas Online Analytics Service (OAS) provides a backend for OAS Dashboards to query the expected visualization.
+
+Both OAS and OAS Dashboards are interdependent services that provides a complete end-to-end Data Analytics solution for actionable insights into the data.
```
Comment (chinmaykolhatkar) [inline docs/oas.md]: Are we going to call superset as OAS Dashboards service?

Comment (lizygeogy) [reply docs/oas.md]: Yes. 
its going to be 
OAS, OAS Dashboards, and CEP Workbench.

**Comment by chinmaykolhatkar** [comment]
@sanjaypujare I've approved the changes. Once you're happy, please merge it.

File: docs/oas.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+- OAS is enabled to stream-in data from Apache Kafka. Any datasource can send the applicable data for analysis into Kafka.
+- The Online Analytics Service takes this real-time feed of data from Apache Kafka, computes, and makes it ready for querying.
+- OAS Dashboards service can query on OAS in real time through DT gateway.
+
+OAS can be specified as a service for an application and managed from the Services page. For more details, refer to [Services](services.md)
+
+# Packaging OAS
+
+Refer to [Packaging Services](services) for more details.
```
Comment (wvary) [inline docs/oas.md]: @lizygeogy Please change [Packaging Services](services) to [Packaging Services](services.md)

File: docs/oas.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+# Managing OAS Service
+
+Refer to [Managing Services](services) for more details.
```
Comment (wvary) [inline docs/oas.md]: @lizygeogy Please change [Managing Services](services) to [Managing Services](services.md)

**Comment by lizygeogy** [comment]
Thank you @wvary. I have made the changes.

**Comment by wvary** [comment]
@lizygeogy I see that this PR is updating the oas.md file.  Where is it used in the documentation?  How can we verify it in mkdocs if it isn't referenced in the mkdocs.yml?

**Comment by lizygeogy** [comment]
@wvary I had not referenced it then, since it had to be within the Services heading where CEP workbench and Superset documents were to be bunched up. These two documents were with the UI team and were not yet merged with Master and I did not know the plan to index these items.

**Comment by lizygeogy** [comment]
@wvary @sanjaypujare  I am adding the index entry for oas.md through a separate PR.

File: docs/oas.md (hunk 1 above)
Comment (sanjaypujare) [reply docs/oas.md]: interdependent services that provides a => interdependent services that provide a

File: docs/oas.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+Online Analytics Service (OAS) is an Apex application which is delivered as a service in some of the DT RTS applications, for example, Omni Channel Fraud Prevention Application.  It is a Druid based system that can query in real-time on data streams from various source applications.
+
+OAS is integrated with OAS Dashboards service which is a DataTorrent RTS Service that has been built using Apache Superset. See [OAS Dashboards Service](oas_dashboards)
+
+OAS Dashboards performs visualization on the frontend, whereas Online Analytics Service (OAS) provides a backend for OAS Dashboards to query the expected visualization.
+
+Both OAS and OAS Dashboards are interdependent services that provides a complete end-to-end Data Analytics solution for actionable insights into the data.
+
+OAS is available only with _DT Premium_ license.
```
Comment (sanjaypujare) [inline docs/oas.md]: Can we provide a link to "DT Premium" license?

File: docs/oas.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+The following image depicts the workflow of OAS.
+
+![](images/Workflow.png)
+
+- OAS is enabled to stream-in data from Apache Kafka. Any datasource can send the applicable data for analysis into Kafka.
+- The Online Analytics Service takes this real-time feed of data from Apache Kafka, computes, and makes it ready for querying.
+- OAS Dashboards service can query on OAS in real time through DT gateway.
+
+OAS can be specified as a service for an application and managed from the Services page. For more details, refer to [Services](services.md)
+
```
Comment (sanjaypujare) [inline docs/oas.md]: Wouldn't it be helpful to describe interfaces to OAS such as:

- Kafka message bus details (broker list and topic(s))
- OAS query API details - connection info and how to use the API

File: docs/oas.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+OAS is available only with _DT Premium_ license.
+
+# Workflow of OAS
+
+The following image depicts the workflow of OAS.
+
+![](images/Workflow.png)
+
+- OAS is enabled to stream-in data from Apache Kafka. Any datasource can send the applicable data for analysis into Kafka.
+- The Online Analytics Service takes this real-time feed of data from Apache Kafka, computes, and makes it ready for querying.
```
Comment (sanjaypujare) [inline docs/oas.md]: "computes" => can we briefly describe the processing/computation done by the service?

**Comment by sashadt** [comment]
@sanjaypujare Can you please add the following reference to `mkdocs.yml` to this PR:

```yml
- Services: 
  ...
  OAS - Online Analytics Service: oas.md
  ...
```

**Comment by sanjaypujare** [comment]
@sashadt  done. This is ready to be merged. Can you review and merge it?

**Pull request merged**